avocado/core/__init__.py

```diff
@@ -15,16 +15,12 @@
 
 from .dispatcher import InitDispatcher
 from .future.settings import settings as future_settings
-from .output import BUILTIN_STREAMS, BUILTIN_STREAM_SETS
 
 
 def register_core_options():
-    streams = (['"%s": %s' % _ for _ in BUILTIN_STREAMS.items()] +
-               ['"%s": %s' % _ for _ in BUILTIN_STREAM_SETS.items()])
-    streams = "; ".join(streams)
     help_msg = ("List of comma separated builtin logs, or logging streams "
-                "optionally followed by LEVEL (DEBUG,INFO,...). Builtin "
-                "streams are: %s. By default: 'app'" % streams)
+                "optionally followed by LEVEL (DEBUG,INFO,...). See "
+                "documentation for detailed explanation.Default: 'app'")
```

Comment:
I think it's better to not regress this feature, as it's simple to work around this ordering issue (and there are bonus points for even "better refactored" code).  See https://github.com/clebergnu/avocado/commit/9d07dd0e0565735c5312c2a1cdf75e2a8d20575f

Reply:
Ok @clebergnu,  I just force-pushed here with your suggestion. Let me know if you need a v2 or this is ok.